Makefile

```diff
@@ -14,7 +14,7 @@ include $(addprefix ./vendor/github.com/openshift/library-go/alpha-build-machine
 # $2 - Dockerfile path
 # $3 - context directory for image build
 # It will generate target "image-$(1)" for builing the image an binding it as a prerequisite to target "images".
-$(call build-image,origin-$(GO_PACKAGE),./Dockerfile,.)
+$(call build-image,ocp-cluster-config-operator,registry.svc.ci.openshift.org/ocp/4.2:cluster-config-operator,./Dockerfile,.)
```

Comment:
/cc @tnozicka 
is this correct? Changing the `origin-xxx` to `ocp-xxx` as the examples in https://github.com/openshift/library-go/pull/496

Reply:
You should also change "FROM :4.0" in the rhel images and drop the origin one as there is no OKD yet. 

Might worth to do a subfolder for images but I'll leave it up on you.

I think https://github.com/openshift/oc is pretty updated as something to base it on but otherwise it looks good